Avoid remount of forceRender tab content components on tab swtich in a RouterTab that is inside a Stack (#1439)
vivid-planet/comet: .changeset/breezy-scissors-warn.md

---
"@comet/admin": patch
---

> Avoid remount of forceRender tab ccntent components on tab swtich in a RouterTab that is inside a Stack

Reviewer: ```suggestion
Avoid remount of `RouterTab` with `forceRender={true}` when `RouterTabs` are used inside a `Stack`
```
